packages/main/src/logging/logger.ts

```diff
@@ -1,13 +1,22 @@
 import { App } from '@/app-globals';
-import logger, {
-  type LogFunctions,
-  type LogLevel,
-  type MainErrorHandlerOptions,
-} from 'electron-log'; // eslint-disable-line no-restricted-imports
+import type {
+  LogFunctions,
+  LogLevel,
+  MainErrorHandlerOptions,
+} from 'electron-log';
+import log from 'electron-log/main';
 import fs from 'fs';
 import { EOL } from 'os';
 import path from 'path';
 
+log.initialize();
+
+// This will transport the logs to the renderer process (DevTools) in production too
+log.transports.ipc.level = log.transports.file.level;
+
+Object.assign(console, log.functions);
+const logger = log.scope('main');
```

Comment:
- `initialize`: Get all logs from the renderer into the main logger (the renderer can't write to file)
- `transport.ipc` defaults to `false` for production. Since our users learned to debug the problem with the DevTools, I think it will help to send the main logs into the renderer console.
- `Object.assign(console, log.functions)` to catch also the `console.log`a